xnmt/input.py

```diff
@@ -173,13 +181,54 @@ def count_sents(self, filename):
     npzFile.close()
     return l
 
+class ContVecNPYReader(InputReader, Serializable):
```

Comment:
so this isn't needed anymore?

This may reappear because of the auto merge, so shall I remove the whole class?